### ISSUE TITLE
Ability to use a mocharc config file.

### ISSUE THREAD

**Issue by jamesopti** [sysgears/mochapack#9]
Is there any support for using a mocharc file instead of mochapack.opts?

As of Mocha 6, [mocha.opts is considered legacy](https://mochajs.org/#mochaopts) and I find it easier to specify multiple configs in JSON instead of as command line arguments.

P.S. - Thanks for picking up support for this package!

**Comment by kierans** [comment]
@jamesopti Finding this issue helped me stop pulling my hair out!

However I'm not having success with a `mocha.opts` with a `--require` flag in it. Where should the `mocha.opts` go in your source tree? What's the best way to `--require` a file from the source tree eg:

```
--require ../../src/globals
```

(importing the module in the test file works, but I want to move the require to the `mocha.opts`)

**Comment by Jack-Barry** [comment]
@jamesopti or @kierans can you give this a go with `2.0.0`? If it works and you do not run into the issue opened in #65 then we can mark this as resolved.

Thanks

**Comment by lakeshadow0** [comment]
I am using 2.0.2 and my `.mocharc.yaml` is not being honored in any fashion.

**Comment by Jack-Barry** [comment]
Have you tried it with the `config` flag pointing to your `.mocharc`?

**Comment by lakeshadow0** [comment]
Ah, I see. I just tried it, and got the error `Invalid JSON config file: .mocharc.yaml`. I assume then only JSON configurations are supported.

**Comment by Jack-Barry** [comment]
There might be something out of whack with [`src/cli/argsParser/optionsFromParsedArgs/mocha/mochaOptionsFromParsedArgs.ts`](https://github.com/sysgears/mochapack/blob/master/src/cli/argsParser/optionsFromParsedArgs/mocha/mochaOptionsFromParsedArgs.ts) or [`src/cli/argsParser/optionsFromParsedArgs/mocha/mergeMochaConfigWithOptions.ts`](https://github.com/sysgears/mochapack/blob/master/src/cli/argsParser/optionsFromParsedArgs/mocha/mergeMochaConfigWithOptions.ts)

Basically, Mochapack is aiming to offload as much functionality as possible to Mocha, might have missed something in here as far as extracting the config file/determining which parser to use under the hood by Mocha's internals.
